gcp/PERMISSIONS.md

```diff
@@ -75,14 +75,20 @@ Accounts and permissions
 Privilege escalation
 ====================
 
-If you are an operator and you need to assign new permissions to your user in order to make changes in a cluster you will need to attach some IAM roles to your user.
+If you are an operator and you need to assign new permissions to your user in order to make changes in a cluster or if you want to manage Security Command Center settings or security sources, you will need to attach the following IAM roles to your user:
 
-Rake commands:
+* `roles/owner` in the current project.
+* `roles/iam.serviceAccountAdmin` in the current organization.
+* `roles/securitycenter.admin` in the current organization.
 
-  * `rake grant_owner_role` Attaches the owner role to the current user account
-  * `rake revoke_owner_role` Removes the owner role from the current user account
+This can be done with the following rake commands:
 
-  In the case of a developer's dev project:
+* `rake grant_owner_role` attaches the owner role in the current project to the current user account.
+* `rake revoke_owner_role` removes the owner role in the current project from the current user account.
+* `rake grant_super_powers` attaches organization-level super user roles to the current user account.
+* `rake revoke_super_powers` removes organization-level super user roles from the current user account.
```

Comment:
I'm not a big fan of these names -- I doubt I will remember which of these affects which set of permissions.

How about `(grant|revoke)_admin` and `(grant|revoke)_org_admin`? Or `_project_admin` and `_org_admin`?

Reply:
+1 for choosing more specific names. Although I like the term "super_powers", it seems to me more explicit the term `*org_admin` and `*project_admin`.

Reply:
+1 for better, more consistent names... although not sure if we shouldn't keep existing `*_ownre_role` as it's a) pretty descriptive and b) already there :)

Reply:
I also like existing `*_owner_role` task naming, but will follow Tyler's advice for consistency.